### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Fix docs.

#### WHY

image

#### HOW

Checked with https://www.drlinkcheck.com/

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,18 +81,18 @@ Libraries to help developers to build apps following
 the [Sign-In guidelines for Wear OS](https://developer.android.com/training/wearables/design/sign-in)
 .
 
-- [horologist-auth-composables](./auth/composables): composable screens for authentication use
+- [horologist-auth-composables](./auth-composables): composable screens for authentication use
```
Comment (luizgrp) [inline README.md:84]: This is tricky.

If someone is accessing https://github.com/google/horologist, the original link is the one that is correct: https://github.com/google/horologist/tree/main/auth/composables versus https://github.com/google/horologist/tree/main/auth-composables (returns 404).

However, if accessing https://google.github.io/horologist/ the link proposed by the changes is the correct: https://google.github.io/horologist/auth-composables versus https://google.github.io/horologist/auth/composables

**Review by shumelchyk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yschimke) [reply README.md:84]: Should we move it out of the README? have a separate index.md in docs, and trim most of README.md?

Comment (luizgrp) [reply README.md:84]: I think that's a good idea

Comment (yschimke) [reply README.md:84]: Or use absolute links? Maybe simpler.

Comment (luizgrp) [reply README.md:84]: They are pointing to different places at the moment:
- https://github.com/google/horologist/tree/main/auth/composables from README
- https://google.github.io/horologist/auth-composables/ from DOCS

We would have to choose one if using absolute links.

Comment (yschimke) [reply README.md]: OK, going with absolute.

**Review by luizgrp — APPROVED** [review]

**Pull request merged**